src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/EditingTargetCodeExecution.java

```diff
@@ -340,9 +349,29 @@ private boolean executeLatex()
       return true;
    }
    
+   private boolean executeInlineChunk()
+   {
+      if (!docDisplay_.getSelection().isEmpty())
+         return false;
+      
+      Token token = docDisplay_.getTokenAt(docDisplay_.getCursorPosition());
```

Comment:
This probably needs a null check -- I think that this routine will return `null` for empty lines. (I realize this was code written by me so technically it's my bug!)